audio/engine.asm

```diff
@@ -1356,6 +1379,7 @@ ParseMusicCommand:
 	ld d, 0
 	; seek command pointer
 	ld hl, MusicCommands
+	; each command takes up two bytes
```

Comment:
This is pretty much implied by the fact that the command offset is doubled before being added to the pointer. Labelling all such indexing operations with a comment would create lots of noise; it's a very common pattern.